csrc/transformer/inference/csrc/layer_norm.cu

```diff
@@ -200,7 +200,11 @@ Template arg:
     StoreResidual: controls whether the residual calculation is stored
         or not. When set to false, the input `res_output` is unused.
 */
-template <typename T, int UNROLL, bool StoreResidual>
+template <typename T, int UNROLL,
```

Comment:
Can we fold `internal_unroll` and `UNROLL` into a single parameter? They're always used as a pair in the implementation as far as I can tell.